test/integration/router_test.go

```diff
@@ -610,6 +626,9 @@ func TestRouterPathSpecificity(t *testing.T) {
 	if err := waitForRoute(routeAddress, "www.example.com", "http", nil, tr.HelloPodAlternate); err != nil {
 		t.Fatalf("unexpected response: %q", err)
 	}
+	if err := waitForRoute(routeTestAddress, "www.example.com:80", "http", nil, tr.HelloPodPath); err != nil {
```

Comment:
Expected response here is slightly different and we should expect `tr.HelloPodAlternate`.